src/components/drinkingCard/DrinkingCard.css

```diff
@@ -0,0 +1,68 @@
+#card-container{
```

Comment:
Kortet er veldig lite på mobil 
![image](https://github.com/user-attachments/assets/665fe1e0-eab4-40aa-879e-9376eb730870)

Reply:
Ja, den er for liten for mobil. Tanken var at neste issue egentlig bare skal slå sammen Joels gridsystem (som skal være optimert også for mobil) med stylingen jeg gjorde. Kanskje dette er dårlig praksis.......